config/jobs/kubernetes-sigs/kind/kind.yaml

```diff
@@ -27,7 +35,7 @@ periodics:
 # conformance test against kubernetes master branch with `kind`
 - interval: 1h
   name: ci-kubernetes-kind-conformance
-  cluster: k8s-infra-prow-build
+  cluster: eks-prow-build-cluster
```

Comment:
Check the cgroup v2 for EKS  before change it?
related: 
- https://github.com/kubernetes/k8s.io/issues/5276

cc @BenTheElder

Reply:
Maybe we should migrate it on the next time.